lib/tentacat.ex

```diff
@@ -56,9 +56,9 @@ defmodule Tentacat do
     _request(:put, url(client, path), client.auth, body)
   end
 
-  def get(path, client, params \\ []) do
+  def get(path, client, params \\ [], encode \\ true) do
```

Comment:
Can we make this an option as a keyword list?

Something like

```def get(path, client, params \\ [], options \\ [])

Keyword.get(options, :encode, true)
```

It would be clearer than just passing true/false